### PR TITLE
Fix teamcity ci

### PR DESCRIPTION
**Pull request by typhoonzero**
1. use dill to serialize model metas
2. default do not use evaluator unless set `train.num_evaluator=1`

### CODE REVIEW

File: pkg/sql/codegen/pai/template_tf.go [hunk 1]
```diff
@@ -82,6 +82,7 @@ model.save_metas("{{.OSSModelDir}}",
 `
 
 const paiRequirementsTmplText = `
+dill
```
Comment (Yancey0623) [inline pkg/sql/codegen/pai/template_tf.go]: using a fixed version?

File: python/sqlflow_submitter/pai/model.py [hunk 2]
```diff
@@ -92,7 +94,7 @@ def save_metas(oss_model_dir, num_workers, file_name, *meta):
             return
     oss_path = get_oss_path_from_uri(oss_model_dir, file_name)
     with gfile.GFile(oss_path, mode='w') as f:
-        pickle.dump(list(meta), f)
+        dill.dump(list(meta), f)
```
Comment (Yancey0623) [inline python/sqlflow_submitter/pai/model.py:97]: Just confusion why changing to `dill`?

Comment (typhoonzero) [reply python/sqlflow_submitter/pai/model.py:97]: Checkout comments under save_metas

**Review by Yancey0623 — APPROVED** [review]
LGTM

**Pull request merged**